Userland/Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -328,9 +333,14 @@ ALWAYS_INLINE ExecutionResult OpCode_CheckEnd::execute(MatchInput const& input,
         if (state.string_position == input.view.length())
             return true;
 
+        // U+2028 LINE SEPARATOR
+        constexpr static u32 const LineSeparator { 0x2028 };
+        // U+2029 PARAGRAPH SEPARATOR
+        constexpr static u32 const ParagraphSeparator { 0x2029 };
```

Comment:
wait why are these repeated here if you already moved them into the global scope?

Reply:
I think I just forgot to remove this set. oops.